packages/configuration-backend/src/service.ts

```diff
@@ -178,3 +178,25 @@ export const validateAdminApiKey = async (p: {
   const { apiKey } = p
   return validateApiKey({ namespace: 'admin', apiKey })
 }
+
+export const getSubscriptionTermsOfServiceBinary = async (p: {
+  namespace: string
+}): Promise<string> => {
+  const { namespace } = p
+  try {
+    const configuration = await getPublicServiceConfiguration({
+      namespace,
+      key: 'merchantSubscriptionTermsOfServiceUrl',
```

Comment:
Could these keys come from some sort of enum? I have liked it more in Java if these keys come from constants/enums. I think that makes these configurationKeys to be found easier. What do you think about this?

Reply:
They could, typescript has enums